### PR TITLE
Update source_installation.rst - Snap packages

### PR DESCRIPTION
**Pull request by cyberanca**
Added instructions on installing Nextcloud via Snap Packages.

cc/ @jospoortvliet @karlitschek

### CODE REVIEW

File: admin_manual/installation/source_installation.rst [hunk 1]
```diff
@@ -22,20 +22,33 @@ Apache and MariaDB, using `the Nextcloud .tar archive
...
    Nextcloud. See :ref:`selinux_tips_label` for a suggested configuration.
 
+.. _snaps_label:
+
+Installing via Snap Packages
+-------------------------------------
+
+A snap is a zip file containing an application together with its dependencies, and a description of how it should safely be run on your system, especially the different ways it should talk to other software. Most importantly snaps are designed to be secure, sandboxed, containerised applications isolated from the underlying system and from other applications. 
+    
+To install the Nextcloud Snap Package (available on Ubuntu 16.04 LTS and above), run the following command in a terminal::
```
Comment (oparoz) [inline admin_manual/installation/source_installation.rst]: Maybe remove the mention of Ubuntu as snapd technology is available on different distros?

**Comment by karlitschek** [comment]
👍

File: admin_manual/installation/source_installation.rst [hunk 2]
```diff
@@ -22,20 +22,33 @@ Apache and MariaDB, using `the Nextcloud .tar archive
...
--------------
+Prerequisites for Manual Installation
+---------------------------------------------
 
-The Nextcloud .tar archive contains all of the required PHP modules. This 
-section lists all required and optional PHP modules.  Consult the `PHP manual 
-<http://php.net/manual/en/extensions.php>`_ for more information on modules. 
-Your Linux distribution should have packages for all required modules. You can 
-check the presence of a module by typing ``php -m | grep -i <module_name>``. 
+The Nextcloud .tar archive contains all of the required PHP modules. This
```
Comment (tflidd) [inline admin_manual/installation/source_installation.rst]: This is not clear, when all modules are contained in the .tar archive, why do I need a list of required and optional modules? Does the tar-archive contain any php module?

**Comment by MorrisJobke** [comment]
I fixed the conflict and merged it via 311cdbf0

**Comment by MorrisJobke** [comment]
Thanks for your help, @cyberanca and sorry that this took that long ... I simply forgot about it.

**Pull request closed without merging**